orange-datamodel/src/test/resources/minimally.populated.orange.json

```diff
@@ -316,7 +316,15 @@
         "maxCopyNumber": 1.2,
         "minMinorAlleleCopyNumber": 0.4
       }
-    ]
+    ],
+    "tumorStats": {
```

Comment:
I don't think these should be (all) 0? There is for example a hotspot mutation in this json.